.github/workflows/test-environment.yml

```diff
@@ -36,6 +36,7 @@ env:
   AWS_REGION: "eu-west-1"
   WORKING_DIR: deploy/test-environments
   FLEET_API_DIR: fleet_api/src
+  AWS_DEFAULT_TAGS: "Key=division,Value=engineering Key=org,Value=security Key=team,Value=cloud-security-posture Key=project,Value=test-environments"
```

Comment:
Is there a point in keeping this here if it's the same as the default justfile command value? I think it's a good practice to have our CI running with the default justfile values, so if someone breaks some value there it will fail the CI (obviously not in this case, since it will only affect the tags)

Reply:
In CI, we don't use the default value for the tags in `just create-cnvm-stack-tags` because we include the dynamically retrieved `owner` tag `github.actor`.